### PR TITLE
Add finitequiz configuration

### PR DESCRIPTION
**Pull request by LeviUniAcc**
Closes Gamify-IT/issues#250

### CODE REVIEW

**Review by Aaron2000119 — APPROVED** [review]

File: src/config.ts [hunk 1]
```diff
@@ -2,5 +2,5 @@ export default {
   // URL of the RESTful API
   apiBaseUrl: "/overworld/api/v1",
   apiChickenshockUrl: "/minigames/chickenshock/api/v1",
-  mapCommitHash: "3616da6c60dcbb116e91d08c59d79b200b3b91a6",
+  apiFinitequizUrl: "/minigames/finitequiz/api/v1",
```
Comment (maexled) [inline src/config.ts:5]:
```suggestion
  apiFinitequizUrl: "/minigames/finitequiz/api/v1",
  mapCommitHash: "3616da6c60dcbb116e91d08c59d79b200b3b91a6",
```

**Review by maexled — APPROVED** [review]

**Review by delvh — CHANGES_REQUESTED** [review]
Blocking until #17 is merged.
Apart from that, there are a few things I mentioned there that also apply to this branch:
1. the switch on the enum instead of string comparisons
2. mentioning the supported versions explicitly
3. the missing error feedback ([    .catch((error) => {
      toast.error("There was an error updating the minigame configuration.");
      console.log(error);
    });](https://github.com/Gamify-IT/lecturer-interface/blob/fca4fbfa5fd2f8dd42275e75e1363b29c50f6aa5/src/components/EditCrosswordpuzzleModal.vue#L112-L115))

**Comment by maexled** [comment]
> Blocking until #17 is merged. Apart from that, there are a few things I mentioned there that also apply to this branch:
> 
>     1. the switch on the enum instead of string comparisons
> 
>     2. mentioning the supported versions explicitly
> 
>     3. the missing error feedback ([    .catch((error) => {
>        toast.error("There was an error updating the minigame configuration.");
>        console.log(error);
>        });](https://github.com/Gamify-IT/lecturer-interface/blob/fca4fbfa5fd2f8dd42275e75e1363b29c50f6aa5/src/components/EditCrosswordpuzzleModal.vue#L112-L115))

2+3 done. To do 1. we have to merge the crosswordpuzzle configuration branch into this. 
Either we merge the crosswordpuzzle pr first and have the merge conflict here or we merge this pr first and then have the merge conflict in the other pr. I would change this where the merge conflict occurred.

**Review by delvh — APPROVED** [review]
Can be merged once the conflicts are resolved.

**Comment by maexled** [comment]
Now everything should be resolved.

**Pull request merged**